crates/llm-base/src/loader.rs

```diff
@@ -40,6 +40,11 @@ impl From<FileType> for i32 {
                 FileTypeFormat::MostlyQ8_0 => 7,
                 FileTypeFormat::MostlyQ5_0 => 8,
                 FileTypeFormat::MostlyQ5_1 => 9,
```

Comment:
@philpax is there a reason for not reusing the constants defined in `ggml.h` here?

Reply:
I think the reason was convenience, we should swap them with the constants, similarly on how it's done on the [Type](https://github.com/rustformers/llm/blob/main/crates/ggml/src/lib.rs#L126C12-L187) enum.

Reply:
I think I see now why it was not trivial:
The corresponding constants are defined in [`crates/ggml/sys`](https://github.com/rustformers/llm/tree/main/crates/ggml/sys), which is a nested crate inside [`crates/ggml`](https://github.com/rustformers/llm/tree/main/crates/ggml).
At least according to GPT-4, one would have to add `crates/ggml/sys` as a direct dependency to `llm-base`, which might not be desired?

Reply:
Maybe we could move the enums into `crates/ggml` to avoid this?

Reply:
That would be a great idea and was what I was going to do (exposing the `sys` crate so that the enum could be defined in terms of those values), but...

they're actually different in `llama.cpp` 🤦 

https://github.com/ggerganov/llama.cpp/blob/7487137227eb32ed9b12156338b865cb29b2dfd1/llama.h#L96-L116

That also explains why q4_k_m doesn't work right now. Let me see if I can work around this somehow.